### ISSUE TITLE
Does BlackberryAccessibility API still exist?

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#12]
Blackberry OS was discontinued in 2013 and support for their devices ended in 2016. Blackberry sold off their phone division and the newer ones use Android. (I'm not sure if anyone even still makes phones under the brand.)

I wonder if we can drop this, or at least come up with an obsolete indicator?

**Comment by mattgarrish** [comment]
Or perhaps we should add a note that newer versions of Blackberry run on Android so the AndroidAccessibility API will be the correct API to specify in these cases.
